tools/search/recreate_index.sh

```diff
@@ -0,0 +1,28 @@
+#! /bin/bash
+set -euo pipefail
+
+export MASTER_KEY=$(cat /run/secrets/meilisearch)
```

Comment:
Assuming something like https://docs.docker.com/engine/swarm/secrets/, but nobody uses Docker Swarm :)

For Cloud Run, we should be able to use https://cloud.google.com/run/docs/configuring/services/secrets

Reply:
I currently have cloud-run talking to [Google Cloud Secrete Manager](https://cloud.google.com/security/products/secret-manager?hl=en)

In this setup using github actions, what I don't think I've quite figured out is how to get the master key into github build process yet.  A quick google search came up with this

https://github.com/google-github-actions/get-secretmanager-secrets

Maybe that's the last piece of it?

Reply:
Wait...I'm not thinking this through well enough.

The github action is meant as build smoke test? Cause the "real" deployment build is a weekly reindex which we would just trigger inside Google Cloud itself? I think?